docs/api-testing/project-access.md

```diff
@@ -1,66 +1,85 @@
 
-### Access
+- **Team Access**: Available if an organization admin has set up teams in Team Management. Users must be a member of the assigned team or an organization admin to access the project. 
+- **Organization Access**: Users who are a member of any team and organization admins can access the project. Individual users have access only if they created the project. 
 
-Access is assigned when you create a project. You can set the project to be accessible to your team (if you're part of one) or to the entire organization.
+The type of project access users have and can configure for the projects they create is based on the following user scenarios:
 
-<img src={useBaseUrl('img/api-fortress/2021/12/accessOrg1.png')} alt="New user UI" width="300" />
+- **Individual User**: Individual users who are not a member of a team. When these users first log in to API Testing, they will be guided through creating a project and a test.<br/><img src={useBaseUrl('img/api-fortress/2021/12/emptyState.png')} alt="New user UI" width="600" /><br/> When creating a project, access will default to organization-wide access, and individual users will only be able to access the projects they've created.
+- **Team Member**: Users who are a member of a team. They can work in projects with access assigned to the team that they are a member of and also projects with organization-wide access. They cannot view projects with access assigned to teams they are not a member of. When creating a project, they will have the option to set the project access to the team that they are a member of or organization-wide access. 
+- **Organization Admin**: Organization Admins can create and interact with all data in all projects across their organization.
 
-You cannot edit project access once it has been set.
+:::tip
+You can view the access set for projects on the Projects page.<br/><img src={useBaseUrl('img/api-fortress/2021/12/accessOrg2.png')} alt="New user UI" width="500" />
+:::
 
-#### User Scenarios
+## Create a Project
 
-- **New User** (no team assignments or projects): When this user first logs in to API Testing, they will be guided through creating a project and a test.<br/><img src={useBaseUrl('img/api-fortress/2021/12/emptyState.png')} alt="New user UI" width="500" /><br/> They will have the option to set organization-wide project access and will only be able to see the projects they created.
-- **Team Member**: This user belongs to a team and can create and interact with their team's projects and organization-wide projects. They cannot view projects with access restricted to an outside team. When creating a project, they will have the option to set access to be team-specific and organization-wide.
-- **Organization Admin**: This user can create and interact with all data in all projects across their organization.
+You create projects and edit project settings on the **Settings > General > Project details** page.  
```

Comment:
@aliciakrebs it's wrong! you cannot create projects in Settings. The only way to create a project is from the Project (list) page (the first screen you reach out coming from SL dashboard)